fperms_iscore/forms.py

```diff
@@ -0,0 +1,79 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext
+from django.utils.translation import ugettext_lazy as _
+
+from fperms.conf import settings as fperms_settings
+from fperms.models import Perm, Group
+
+from is_core.forms.models import SmartModelForm, ModelMultipleChoiceField
+
+
+def max_or_zero(*values):
+    return max(0, 0, *values)
```

Comment:
I had to google why you used twice the zero value, but this is the shortest solution, that's true :(